service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -86,7 +83,8 @@ public void createNewUser(String username, String email, String password, String
      * Given a JWT signature, store it in the BLACKLISTED_REFRESHES table.
      */
     public void addToBlackList(String signature) {
-        Timestamp expirationTimestamp = Timestamp.from(Instant.now().plusMillis(AuthUtils.refresh_exp));
+        Timestamp expirationTimestamp = Timestamp.from(Instant.now().plusMillis(
+            Long.valueOf(PropertiesLoader.getExpirationProperties().getProperty("days_refresh_expiration"))));
```

Comment:
As a general rule we probably shouldn't have any in-line property fetching, just break it up into a separate variable first

Reply:
see if you like what i did